### ISSUE TITLE
Minor bug on UI - Tooltip shows for a hidden element

### ISSUE THREAD

**Issue by KevinAsher** [cypress-io/cypress#22717]
### Current behavior

While playing with the built-in demos that came with the installation of cypress, I've noticed a tooltip that keeps showing up for a hidden error icon as shown in the image below. The tooltip only shows up if you position your mouse pointer above the hidden icon:

![image](https://user-images.githubusercontent.com/20466664/177981523-12a9a172-2a5d-460d-ad27-59aed1c6b111.png)

### Desired behavior

The tooltip shouldn't appear when the icon is hidden since there wasn't any error on the tests.

### Test code to reproduce

Can be reproduced with the example E2E tests that came with default cypress installation.

### Cypress Version

10.3.0

**Comment by calummcgowan** [comment]
Hello, I'm looking to do some open-source work and I can see that this is marked good first issue. Can I pick up this ticket?

**Comment by jennifer-shehane** [comment]
@calummcgowan Yes, please see our Contributing.md guide in the base of the repo

**Comment by calummcgowan** [comment]
@jennifer-shehane Hi there. I'm following the contributing.md file, but I'm having some issues getting it to run. Do you happen to know how I might resolve these errors? The links provided do not offer much help. Thanks!
![Screenshot 2024-07-12 132723](https://github.com/user-attachments/assets/eec5f23c-05a4-4adf-b22c-c88e9451c08a)
